### PR TITLE
Chore: be consistent between files regarding spaces and alignements

### PR DESCRIPTION
**Pull request by lespacedunmatin**
Purely presentationnal PR.

### CODE REVIEW

File: src/config/steps/fing.coffee [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-  name: 'fing',
-  view : 'steps/fing'
-  isActive: (instance) ->
-    instance.attributes \
-      && instance.attributes.context \
-      && instance.attributes.context.toLowerCase() is 'fing'
+    name: 'fing',
+    view : 'steps/fing'
+    isActive: (instance) ->
+        instance.attributes \
```
Comment (gregorylegarec) [inline src/config/steps/fing.coffee:5]: The two following lines should also be indented, as they are the continuity of the first one, like this :
```js
    ...
    isActive: (instance) ->
        instance.attributes \
            && instance.attributes.context \
            && instance.attributes.context.tolowerCase() is 'fing'
```

File: src/config/steps/maif.coffee [hunk 2]
```diff
@@ -2,7 +2,7 @@ module.exports = {
     name: 'maif',
     view : 'steps/maif'
     isActive: (instance) ->
-      instance.attributes \
+        instance.attributes \
```
Comment (gregorylegarec) [inline src/config/steps/maif.coffee:5]: Same remark below this line.

**Review by gregorylegarec — CHANGES_REQUESTED** [review]
Thanks @lespacedunmatin, looks ok 👍 
Just a change request about indentation of "splitted lines".

**Review by gregorylegarec — APPROVED** [review]

**Comment by gregorylegarec** [comment]
Thanks @lespacedunmatin

File: src/config/steps/fing.coffee (hunk 1 above)
Comment (lespacedunmatin) [reply src/config/steps/fing.coffee:5]: I do not understand that way of indenting code, to me it leads to poorly indented code like: 
```
if (
  toto
    && tata
) {
// do stuff
}
```

Comment (gregorylegarec) [reply src/config/steps/fing.coffee:5]: @lespacedunmatin Sorry I missed your comment. The thing is that the whole app is indented this way and we need to keep consistency.

**Pull request merged**